### DISCUSSION TITLE
How to build PBPK/PD models?

### DISCUSSION THREAD

**Discussion by Gre-ux** [Open-Systems-Pharmacology/Forum#1690] [How-Tos] [answered]
Hi,
I am working on a antidiabetic drug. Now I have it's plasma concentration-time profile and changes in FPG and HbA1c. Is there a way that I could use the PBPK model I built in PK-Sim and transported it to MoBi to build a PBPK/PD model? Is there any tutorial? What PD model is the best fit? Thanks very much for your help!

**Comment by PavelBal** [comment, marked as answer]
You can check out one of our PBPK-PD models on glucose homeostasis and the corresponding publications:

- https://github.com/Open-Systems-Pharmacology/Glucose-Insulin-Model
- https://github.com/Open-Systems-Pharmacology/SGLT2i-hyperfiltration-model
- https://github.com/Open-Systems-Pharmacology/Incretins-Model

**Reply by Gre-ux** [reply]
Thanks very much for your reply!

**Comment by nicospinu** [comment]
At esqLABS, we have a few self-paced tutorials available that could help.

These ones would be in particular relevant for learning about PK-Sim and MoBi interaction:  
- https://esqlabs.shop/products/a-two-container-model-2
- https://esqlabs.shop/products/observer-2
- https://esqlabs.shop/products/formula-types-2

These ones are applied case studies, unfortunately, not in diabetes, which could also help how to use both PK-Sim and MoBi:
- https://esqlabs.shop/products/dermal-absorption-of-caffeine-2
- https://esqlabs.shop/products/maternal-fetal-pbpk-2
- https://esqlabs.shop/products/antibody-therapy-in-oncology-2

If you are working in academia, consider getting free access at this link https://learn-esqlabs.talentlms.com/

If you have any questions, feel free to reach out at training@esqlabs.com

**Reply by Gre-ux** [reply]
Thanks very much for your reply!
